src/DatabaseManager.php

```diff
@@ -85,95 +74,49 @@ public function getConnections()
         ];
     }
 
-    /**
-     * @param string $query
-     * @param array  $params
-     *
-     * @return array
-     */
-    public function fetchAll($query, array $params = [])
+    public function fetchAll(string $query, array $params = []): array
     {
         return $this->getReadAdapter()
             ->fetchAll($query, $params);
     }
 
-    /**
-     * @param string $query
-     * @param array  $params
-     *
-     * @return array
-     */
-    public function fetchOne($query, array $params = [])
+    public function fetchOne(string $query, array $params = []): array
     {
         return $this->getReadAdapter()
             ->fetchOne($query, $params);
     }
 
-    /**
-     * @param string $query
-     * @param array  $params
-     *
-     * @return string
-     */
-    public function fetchValue($query, array $params = [])
+    public function fetchValue(string $query, array $params = [])
```

Comment:
That's very odd, but ok. It really shouldn't be allowed at all at the language level.